pyof/v0x04/controller2switch/packet_out.py

```diff
@@ -35,7 +35,8 @@ class PacketOut(GenericMessage):
     #:    (Only meaningful if buffer_id == -1.)
     data = BinaryData()
 
-    def __init__(self, xid=None, buffer_id=None, in_port=None, actions=None,
+    def __init__(self, xid=None, buffer_id=0xffffffff,
```

Comment:
@renanrodrigo I think we have a `constant` for this buffer_id. It is in `pyof.v0x01.common.constants` (I think we may move it to `pyof.foundation.constants`.

Reply:
I do think it's a better idea to create a new constant in `pyof.v0x04.common.constants` then: many things as those constants are version specific, and moving it to `foundation` may lead to future trouble. Do you agree?

Reply:
If it is a constant that may be version specific, then I do agree.

In this specific case, I may be wrong, but this value is the 'UBInt32 max value' right?

If so, you can either use UBINT32_MAX (`pyof.foundation.constants`) or you can create the `NO_BUFFER` constant on `pyof.v0x04.common.constants` and assing UBINT32_MAX to it. =)
Its your call! ;)

Reply:
Totally right.